tests/src/test/fdc3.raiseIntent.ts

```diff
@@ -1,4 +1,10 @@
-import { Context, raiseIntent, ResolveError } from "@finos/fdc3";
+import {
+  AppMetadata,
+  Context,
+  IntentResolution,
+  raiseIntent,
+  ResolveError,
+} from "@finos/fdc3";
 import { assert, expect } from "chai";
 import APIDocumentation from "../apiDocuments";
 
```

Comment:
createReceiver just below here should probably have an integrated timeout that will reject its promise once hit...